package.json

```diff
@@ -64,10 +66,18 @@
     "build": "react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
-    "lint": "eslint '*/**/*.{js,ts,tsx}' --quiet --fix",
+    "type:check": "tsc",
+    "lint": "eslint */**/*.{js,ts,tsx}",
+    "lint:fix": "eslint --fix */**/*.{jsx,js,tsx}",
+    "prettier": "prettier . --ignore-path .gitignore",
+    "format": "prettier --write",
```

Comment:
I think `npm run format` is superfluous when `npm run format:fix` is also defined

Reply:
should I do the same thingfor the `npm run lint` is superfluous when `npm run lint:fix` too?

Reply:
I'll work on condensing those commands. Because the `format` and `lint` targets the files in staging and `fix` and `check` scripts were to target global linting and formatting

Reply:
Hi @JulianaSau, I am just seeing your comment. can you help me check if I have made all the changes as they should be, then correct or fix any if necessary and resolve them?